extensions/amp-story-interactive/0.1/test/test-amp-story-interactive-binary-poll.js

```diff
@@ -44,6 +45,7 @@ describes.realWin(
       const ampStoryPollEl = win.document.createElement(
         'amp-story-interactive-binary-poll'
       );
+      ampStoryPollEl.getAmpDoc = () => new AmpDocSingle(win);
```

Comment:
If we use `this.getAmpDoc()`, we might not need to mock this (or at least mock it directly on the `AmpStoryInteractiveBinaryPoll` instead of it's element)